README.md

```diff
@@ -23,6 +23,11 @@ Comments, suggestions, and improvements are welcome and appreciated.
 
 + [x'=x^2](http://nbviewer.jupyter.org/github/PerezHz/TaylorIntegration.jl/blob/master/examples/x-dot-equals-x-squared.ipynb)
 + [Kepler problem](http://nbviewer.jupyter.org/github/PerezHz/TaylorIntegration.jl/blob/master/examples/Kepler-problem.ipynb)
++ [Evaluating definite integrals](http://nbviewer.jupyter.org/github/PerezHz/TaylorIntegration\
+.jl/blob/master/examples/Evaluating-definite-integrals.ipynb)
++ [Damped, driven linear oscillator](http://nbviewer.jupyter.org/github/PerezHz/Ta\
+ylorIntegration\
+.jl/blob/master/examples/Damped-driven-linear-oscillator.ipynb)
 
```

Comment:
Sure, I agree! Actually it should be already listed in the examples section of `README.md`